From 315f2c8cad43668df5154659c7d8856fb8ed0c0e Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sun, 2 Aug 2026 13:38:34 +0200
Subject: [PATCH] ARM: dts: allwinner: sun8i-a83t-tbs-a711: Add touchscreen
 IOVCC supply

The touchscreen's I/O rail is the same gated 3.1V CTP rail that already
feeds its main supply. Without it the driver uses a dummy regulator:

  edt_ft5x06 0-0038: supply iovcc not found, using dummy regulator

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm/boot/dts/allwinner/sun8i-a83t-tbs-a711.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/allwinner/sun8i-a83t-tbs-a711.dts b/arch/arm/boot/dts/allwinner/sun8i-a83t-tbs-a711.dts
index 0d029f57fc02..74730e435bee 100644
--- a/arch/arm/boot/dts/allwinner/sun8i-a83t-tbs-a711.dts
+++ b/arch/arm/boot/dts/allwinner/sun8i-a83t-tbs-a711.dts
@@ -317,6 +317,7 @@ touchscreen@38 {
 		interrupts = <0 7 IRQ_TYPE_EDGE_FALLING>; /* PL7 */
 		reset-gpios = <&pio 3 5 GPIO_ACTIVE_LOW>; /* PD5 */
 		vcc-supply = <&reg_ldo_io0>;
+		iovcc-supply = <&reg_ldo_io0>;
 		touchscreen-size-x = <1024>;
 		touchscreen-size-y = <600>;
 	};
-- 
2.43.0

